src/taco-cli/cli/platform.ts

```diff
@@ -170,6 +176,22 @@ class Platform extends KitComponentCommand {
                 break;
 
         }
+
+    }
+    private list(): Q.Promise<ICommandTelemetryProperties> {
+        return this.passthrough()
```

Comment:
Potentially we could kick off the settings loading as well to reduce load times?
`return Q.all([this.passthrough(), Settings.loadSettingsOrReturnEmpty()]).spread(function (telemetryProperties, settings) { ...`